lib/mix/tasks/bird.gen.docs.ex

```diff
@@ -11,7 +11,11 @@ defmodule Mix.Tasks.Bird.Gen.Docs do
 
   @doc false
   def run(_args) do
-    if System.find_executable("aglio") == nil do
+    aglio_path =
+      Application.get_env(:blue_bird, :aglio_path, "aglio")
+      |> Path.absname()
+
+    if File.exists?(aglio_path) == false do
```

Comment:
Thanks for the PR! Unfortunately, this doesn't work like it is, because the default value `aglio` is not a complete path to the binary and `File.exists?/1` doesn't find it (see ci). Perhaps you could make a check on the `:aglio_path` first: If a value is set, use `File.exists?/1`, and if it is not set, try to find the globally installed executable with `System.find_executable/1`.

What do you think?

@rhazdon

Reply:
Oh, and if you implement it that way, it would be great to have the error message reflect both ways.